src/schema/api/Column/ColumnRestriction.h

```diff
@@ -0,0 +1,33 @@
+#ifndef LOODB_COLUMNRESTRICTION_H
+#define LOODB_COLUMNRESTRICTION_H
+
+#include "Column.h"
+#include "../Entry.h"
+#include "../../../filesystem/pagination/page/api/Page.h"
+#include <memory>
+
+// Private members of Column Restriction
+struct ColumnRestrictionImpl{
+  std::unique_ptr<Column> base_;
+};
+
+// Decorator for Column
+class ColumnRestriction : public Column  {
+
+    // pImpl for Column Restriction
+    std::unique_ptr<ColumnRestrictionImpl> impl_;
+
+    virtual bool valid_(Entry);
+
```

Comment:
Hey I think that these virtual override functions should be labelled just "override"

https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rh-override

Reply:
Good call!